sphinx/ext/imgconverter.py

```diff
@@ -30,9 +30,13 @@ def is_available(self) -> bool:
             subprocess.run(args, stdout=PIPE, stderr=PIPE, check=True)
             return True
         except OSError as exc:
-            logger.warning(__('convert command %r cannot be run, '
-                              'check the image_converter setting: %s'),
-                           self.config.image_converter, exc)
+            logger.warning(__(
+                f"Unable to run the image conversion command {self.config.image_converter!r}!"
```

Comment:
Usually, we don't use exclamation marks for warning. Could you replace it with a period?